### DISCUSSION TITLE
ERROR EXECUTING YOUR CODE

### DISCUSSION THREAD

**Discussion by andr3scontrer4s** [rramosp/ai4eng.v1#303] [02 - PYTHON]
Hola, buen día

En el lab02.03 estoy teniendo un problema para hacer la entrega del código, me sale el siguiente mensaje

![lab02 03 error](https://github.com/rramosp/ai4eng.v1/assets/164700198/3303c5e0-92a7-400e-b947-4be353062d19)

A que se debe el error? El código que escribí funciona pero no deja submit

**Comment by Felipe-RA** [comment]
Saludos @andr3scontrer4s 

El problema es que el corrector está buscando en un dataframe una columna margin, pero no la logra encontrar.

Verifica que tengas la lógica para manejar los casos en los que la columna `margin` no esten disponibles
